### PR TITLE
feat: add doc for databend

### PR DESCRIPTION
**Pull request by hantmac**

### CODE REVIEW

File: modules/databend/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        return
+    }
+
+    db, err := sql.Open("databend", connectionString)
+    if err != nil {
+        log.Printf("failed to connect to Databend: %s", err)
+        return
+    }
+    defer db.Close()
+    )
```
Comment (mdelapenya) [inline modules/databend/index.md:41]: I think we do not need the entire connection example 🤔

File: modules/databend/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+          <version>1.20.1</version>
+          <scope>test</scope>
+      </dependency>
+      ```
+  - id: go
+    url: https://golang.testcontainers.org/modules/databend/
+    maintainer: core
+    example: |
+      ```go
+    databendContainer, err := databend.Run(ctx, "datafuselabs/databend:v1.2.615", databend.WithUsername("test1"), databend.WithPassword("pass1"),
```
Comment (mdelapenya) [inline modules/databend/index.md:28]: Typo: adding the closing parenthesis
```suggestion
    databendContainer, err := databend.Run(ctx, "datafuselabs/databend:v1.2.615", databend.WithUsername("test1"), databend.WithPassword("pass1"))
```

File: modules/databend/index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+title: Databend
+categories:
+  - relational-database
+docs:
+  - id: java
+    url: https://java.testcontainers.org/modules/databases/databend/
+    maintainer: core
+    example: |
+      ```java
+      var databendContainer = new DatabendContainer();
```
Comment (mdelapenya) [inline modules/databend/index.md:11]: @eddumelendez do we need to explicitly pass the image in the examples?

File: modules/databend/index.md (hunk 1 above)
Comment (hantmac) [reply modules/databend/index.md]: Make sense. Fixed.

File: modules/databend/index.md (hunk 3 above)
Comment (hantmac) [reply modules/databend/index.md]: @mdelapenya We need to pass the image name, I fixed it. Thanks!

**Review by mdelapenya — APPROVED** [review]
LGTM, thanks!

**Comment by mdelapenya** [comment]
@hantmac I've added #76 fixing the white spaces (I tested it locally but never pushed the changes 🤦 )

We also updated the logo to comply with the design rules for the logos. PLMK what you think.

**Pull request merged**